bugbot/utils.py

```diff
@@ -765,3 +765,28 @@ def is_keywords_removed_by_bugbot(bug: dict, keywords: Iterable) -> bool:
         if change["field_name"] == "keywords"
         for keyword in keywords
     )
+
+
+def get_bug_bugdash_url(component, tab_name: str) -> str:
+    """
+    Generate bugdash URL from a component.
+
+    Args:
+        component: The component as a ComponentName object.
```

Comment:
We do not need to describe the type since it is already specified in the type hint:
```suggestion
        component: The name of the targeted component.
```